app/views/users/show.html.erb

```diff
@@ -1,46 +1,42 @@
-
-<div class="img-user-container" >
-                imagen
-</div>
-<div class="info-user-container" >
-    <div>
-        <h2>
-            <%= @user.name%>
-        </h2>
+<div class="list-item">
+    <div class="img-user-container" >
+        <img src= <%= @user.photo%>>
     </div>
-    <div>
-        <p>
+    <div class="info-user-container" >
+        <div>
+            <h2>
+                <%= @user.name%>
+            </h2>
+        </div>
+        <p class="post-number">
             Number of posts: <%= @user.post_counter%>
         </p>
     </div>
 </div>
-<div>
+<div class="bio-container">
     <h3>
         Bio
     </h3>
     <p>
         <%= @user.bio%>
     </p>
 </div>
-<div>
-    <h3>
-        Posts
-    </h3>
-    <ul>
+<div class="post-container">
+    <ul class="post-list">
         <% @user.posts.each do |post| %>
-            <li>
+            <li class="post-item">
                 <h4> <%= post.title%> </h4>
                 <p>
                 <%= post.text %>
                 </p>
-                <p> Comments: <%= post.comments_counter %> - Likes: <%= post.likes_counter%> </p>
+                <p class="post-counters"> Comments: <%= post.comments_counter %> - Likes: <%= post.likes_counter%> </p>
             </li>
         <% end %>
     </ul>
 
-    <div>
-        <button> See all posts 
-            
+    <div class="user-button-container">
+        <button class="btn-see-posts">
+            <%= link_to "See all posts", user_posts_path(@user) %>
         </button>
 
```

Comment:
- [ ] Good job with implementing the majority of the wireframes but a click on the `See all posts` button is leading to some views issues and I believe you can work on that. Kindly fix this and let's celebrate the milestone! :wave: 
![image](https://github.com/ulises2607/Blog-App/assets/99715640/f04cc30f-17c2-4ecd-a26a-3bced770ce8e)